docs/topics/OAuth2.md

```diff
@@ -56,68 +43,232 @@ Scopes provide access to certain resources of a user's account. Your API client
+
+Bot accounts have a few differences in comparison to normal user accounts, namely:
+
+1. Bots are added to servers through the OAuth2 API, and cannot accept normal invites.
+2. Bots cannot have friends, nor be added to or join Group DMs.
+3. Bots do not have a maximum number of Guilds (unlike user accounts, which are limited to 100).
+4. Bots have an entirely separate set of [Rate Limits](#DOCS_RATE_LIMITS/rate-limits).
+
+### Bot Auth Flow
+
+Bot auth is a special server-less and callback-less OAuth2 flow that makes it easy for users to add bots to servers. The URL you create looks similar to what we use for full stack implementation:
 
 ```
-https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=webhook.incoming&redirect_uri=https%3A%2F%2Fnicememe.website&response_type=code
+https://discordapp.com/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=1
 ```
 
-`client_id` is your application's ID and `redirect_uri` is one of your application's URL-encoded redirect URIs.
+In the case of bots, the `scope` parameter should be set to `bot`. There's also a new parameter, `permissions`, which is an integer corresponding to the [permission calculations](#DOCS_PERMISSIONS/permissions-bitwise-permission-flags) for the bot. You'll also notice the absence of `response_type` and `redirect_uri`. Bot auth does not require these parameters because there is no need to retrieve the user's access token.
+
+When the user navigates to this page, they'll be prompted to add the bot to a server in which they have proper permissions. On acceptance, the bot will be added. Super easy!
+
+### Advanced Bot Auth
+
+Enterprising devs can add some complexity to bot auth. You can request additional scopes outside of "bot", which will prompt a continuation into the [authorization code](#DOCS_OAUTH2/authorization-code-grant) flow and add the ability to request the user's access token. If you request any scopes outside of `bot`, the `response_type` and `redirect_uri` parameters are again mandatory. A full OAuth2 flow is also required if you check "Require OAuth2 Code Grant" in your application's settings.
 
-When a user is directed to this URL, they are prompted to select a channel for the webhook to be placed in. Your application will receive an authorization code back in the querystring (as usual with the authorization code grant).
+If you continue into the full stack flow while including "bot" in your scopes, you'll get some additional querystring parameters on redirection: `guild_id`, the id of the guild to which the bot was added, and `permissions`, the permission integer from the original URL.
```

Comment:
> including `bot`

also need to make a note about the security of the data in the querystring: https://github.com/hammerandchisel/discord-api-docs/issues/320#issuecomment-318130230